components/lib/tree/UITreeNode.js

```diff
@@ -374,26 +374,45 @@ export const UITreeNode = React.memo((props) => {
         }
     };
 
+    const findChildrenByKey = (data, key) => {
```

Comment:
Is there any way to move this function into UITreeNode.js or a Utility class it looks like its copied 3 times?